snf-ui/app/index.html

```diff
@@ -20,6 +20,7 @@
   </head>
   <body class="{{content-for 'body-cls'}}">
     {{content-for 'body'}}
+    <script src="assets/ace/ace.js"></script>
```

Comment:
isn't this handled in Brocfile.js? Can you test if the app is functional without the <script> tag?